### DISCUSSION TITLE
How do you enable Jakarta validation on the POJOs? There is no mention of this in the docs.

### DISCUSSION THREAD

**Discussion by aminmansour** [quarkiverse/quarkus-openapi-generator#764] [Q&A] [answered]
No Jakarta validations are being added as part of the generation. I see from the template that useBeanValidation has to be enabled for it to be added, but how can we define useBeanValidation to be set to true.

**Comment by mcruzdev** [comment, marked as answer]
Hi @aminmansour, how are you? Thank you for opening this discussion!

Searching in the code there is no way to configure `useBeanValidation`, this is configured by the parent class `org.openapitools.codegen.languages.JavaClientCodegen`.

@FelixScherzinger is working in a pull request to add it, see more details [here](https://github.com/quarkiverse/quarkus-openapi-generator/pull/746).
